datahub-web-react/src/app/search/SearchBar.tsx

```diff
@@ -411,6 +421,7 @@ export const SearchBar = ({
                         </>
                     }
                     ref={searchInputRef}
+                    suffix={(showCommandK && !isFocused && <CommandK />) || null}
```

Comment:
do you think we need this command k icon? tbh it feels a little busy and maybe not necessary to show for this. we could show something as a tooltip when you hover over the search bar that it's possible to do command K?